app/i18n/en/conf.php

```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'HTML5 notification timeout',
 		),
 		'show_nav_buttons' => 'Show the navigation buttons',
-		'theme' => 'Theme',
+		'theme' => array(
+			'_' => 'Theme',
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
+				'description' => 'This theme is deprecated and will be not available anymore in a <a href=https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme target=_blank>future release of FreshRSS</a>',	// TODO
```

Comment:
```suggestion
				'_' => 'Deprecated',	// TODO
				'description' => 'This theme is deprecated and will be not available anymore in a <a href="https://freshrss.github.io/FreshRSS/en/users/05_Configuration.html#theme" target=_blank>future release of FreshRSS</a>',	// TODO
```

Please add quotes (for all the translations)